com.unity.shadergraph/Editor/Data/Graphs/SerializableVirtualTexture.cs

```diff
@@ -20,21 +20,13 @@ public SerializableVirtualTextureLayer(string name, string refName, Serializable
     }
 
     [Serializable]
-    public sealed class SerializableVirtualTexture : ISerializationCallbackReceiver
+    public sealed class SerializableVirtualTexture
```

Comment:
Is this still used anywhere?

Reply:
Yes, used by ProceduralVT and the VT Property.